pyproject.toml

```diff
@@ -101,5 +102,5 @@ force-sort-within-sections = true
 known-first-party = ["pelican"]
 
 [build-system]
-requires = ["pdm-backend"]
-build-backend = "pdm.backend"
+requires = ["uv_build>=0.8.2,<0.9.0"]
+build-backend = "uv_build"
```

Comment:
Now that the [`uv` build backend has stabilized](https://github.com/astral-sh/uv/releases/tag/0.7.19), I think we can remove the upper bound here as well.